cli/ballerina-packerina/src/test/java/org/ballerinalang/packerina/cmd/BuildCommandTest.java

```diff
@@ -744,6 +744,23 @@ public void testBuildCommandSingleFileWithOutput() throws IOException {
         Path execJar = tmpDir.resolve("sample.jar");
         Assert.assertTrue(Files.exists(execJar), "Check if jar gets created");
     }
+
+    @Test(description = "Test the --skip-tests flag in the build command to ensure it avoids compiling tests")
+    public void testBuildWithSkipTests() throws IOException {
```

Comment:
Why we are throwing IOException from here?

Reply:
Since the output log generated from the build command is read at line 756 and the `readOutput` method used for this throws an IOException.

Reply:
Sorry my bad yor are correct., This comment was for  line 732 method which was already there.